release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -449,9 +460,29 @@ Before this release, if an install plan failed, the subscription condition would
 ===== Indicating resolution conflicts on subscription statuses
 Because dependency resolution treats all components in a namespace as a single unit, if a resolution failure occurs, all subscriptions on the namespace now indicate the error.
 
+[id="ocp-4-9-catalog-image-template"]
+==== Image template for custom catalog sources
+
+To avoid cluster upgrades potentially leaving Operator installations in an unsupported state or without a continued update path, you can enable automatically changing your Operator catalog's index image version as part of cluster upgrades.
+
+Set the `olm.catalogImageTemplate` annotation to your catalog image name and use one or more of the Kubernetes cluster version variables when constructing the template for the image tag.
+
+For more information, see xref:../operators/understanding/olm/olm-understanding-olm.adoc#olm-catalogsource-image-template_olm-understanding-olm[Image template for custom catalog sources].
+
 [id="ocp-4-9-osdk"]
 === Operator development
 
+The following new features and enhancements relate to developing Operators with the Operator SDK.
+
+[id="ocp-4-9-osdk-ha-sno"]
+==== High-availability or single node cluster detection and support
```

Comment:
Technically it seems like "single node cluster" should be "single-node cluster", but I am guessing you're using existing language from other SNO docs here.

Reply:
Yeah was trying to defer to other SNO usage in repo. 🤷‍♂️